linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+arg1=$1
+arg2=$2
+arg3=$3
```

Comment:
please use a meaning variable name

this is a bad practice as it would be hard to add new args in the future.

Reply:
Changes done for the variable names